gcn/notices/core/Localization.schema.json

```diff
@@ -13,25 +13,41 @@
       "type": "number",
       "description": "ICRS declination [deg], utilizes the J2000 epoch and an equatorial coordinate system"
     },
-    "uncertainty_shape": {
-      "enum": ["circle", "ellipse", "rectangle"],
-      "description": "In what manner are the errors in RA and Dec defined - as circular, elliptical, or rectangular shapes",
-      "default": "circle"
-    },
-    "ra_uncertainty": {
-      "type": "array",
-      "maxItems": 2,
-      "description": "Uncertainty in RA [deg] with optional asymmetric errors, measured from the RA in the schema; respective ex: [r], [r1], [l1-, l1+]. For ellipse, ra axis is r1 or semi-major axis."
-    },
-    "dec_uncertainty": {
-      "type": "array",
-      "maxItems": 2,
-      "description": "Uncertainty in Dec [deg] with optional asymmetric errors, measured from the Dec in the schema; respective ex: [r2], [l2-, l2+]. Not reported if circular and Dec axis is r2 or semi-minor axis for ellipse."
-    },
-    "position_angle": {
-      "type": "number",
-      "description": "Anti-clockwise position angle from RA axis [deg]. Used if uncertainty_shape ellipse or rectangle is tilted w.r.t. standard (RA, Dec) coordinate.",
-      "default": 0
+    "error": {
+      "oneOf": [
+        {
+          "type": "object",
+          "properties": {
+            "radius": {
+              "type": "number",
+              "description": "Uncertainty region described as a circle with a radius [deg]."
+            }
+          },
+          "required": ["radius"]
+        },
+        {
+          "type": "object",
+          "properties": {
+            "position_error_a": {
+              "type": "number",
+              "description": "Length of the semi-major axis of the uncertainty ellipse [deg]."
+            },
+            "position_error_b": {
+              "type": "number",
+              "description": "Length of the semi-minor axis of the uncertainty ellipse [deg]."
+            },
+            "position_error_pa": {
+              "type": "number",
+              "description": "Astronomical position angle (measured from North through East) of the major axis of the uncertainty ellipse [deg]."
+            }
+          },
+          "required": [
+            "position_error_a",
+            "position_error_b",
+            "position_error_pa"
+          ]
+        }
+      ]
     },
     "containment_probability": {
```

Comment:
If we go with this approach, containment_probability should also be part of error.